### ISSUE TITLE
select u:model example missing

### ISSUE THREAD

**Issue by nerdoc** [adamghill/django-unicorn#280]
This is more a question how a select works with a model. Can't figure it out.
If you tell me in a few short words, I ca try to make an example and a PR for your docs.
```python
# models.py
class Fruit(models.Model):
    name=CharField(max_length=255)

# components/fruit_select.py
class FruitsView(UnicornView):
    fruits=None
    current_fruit = None
    def mount(self):
        self.fruits = Fruit.objects.all()
```
```html
# templates/unicorn/fruits.html 
<select u:model="fruits">
    {% for fruit in fruits %}
      <option value="{{ fruit.id }}"{% if fruit.id == current_fruit %} selected{% endif %}>{{ fruit }}</option>
    {% endfor %}
</select>
```

**Comment by adamghill** [comment]
The example I have is https://github.com/adamghill/django-unicorn/blob/master/example/unicorn/templates/unicorn/html-inputs.html#L22-L26. Can you try something like this and see if it works as you expect:

```html
# templates/unicorn/fruits.html 
<select u:model="current_fruit">
    {% for fruit in fruits %}
      <option value="{{ fruit.id }}">{{ fruit }}</option>
    {% endfor %}
</select>
```

**Comment by nerdoc** [comment]
Definitely, that works. Could you (or I?) add that to the docs' examples?

**Comment by adamghill** [comment]
Do you mean adding an example like https://www.django-unicorn.com/examples/todo? I can add one there for sure, just let me know if that makes sense to you.

**Comment by nerdoc** [comment]
Yes, that's what I meant.

**Comment by adamghill** [comment]
I added https://www.django-unicorn.com/examples/select in https://github.com/adamghill/django-unicorn.com/commit/da80f91e1f6a8c5c860e94ba210a147d976b61a6. 

Closing this, but let me know if there are other examples that might make sense on the documentation site. I tend to use the `example` site in this repo for all my crazy experiments, but showing fun examples on the doc site would probably be useful.
